Clarify default enabled services (#2552)
canonical/ubuntu-pro-client: docs/howtoguides/get_token_and_attach.md

# How to get an Ubuntu Pro token and attach to a subscription

## Get an Ubuntu Pro token

Retrieve your Ubuntu Pro token from the

> [Ubuntu Pro portal](https://ubuntu.com/pro/). You will log in with your "Single

Reviewer: ```suggestion
[Ubuntu Pro portal](https://ubuntu.com/pro/). Log in with your "Single
```

Once the Ubuntu Pro Client is attached to your Ubuntu Pro account, you can use
it to activate various services, including: access to ESM packages, Livepatch,
FIPS, and CIS. Some features are specific to certain LTS releases.

## Control of auto-enabled services

> Your subscription controls which services are available to you as shown and
> controlled on the [Ubuntu Pro Dashboard](https://ubuntu.com/pro/dashboard).

Reviewer: ```suggestion
Your subscription controls which services are available to you and which ones
you can manage via the [Ubuntu Pro Dashboard](https://ubuntu.com/pro/dashboard).
```

Recommended services are auto-enabled when attaching a system. Which services

> are considered recommended can, for some subscriptions, be controlled on the

Reviewer: I'm not sure about this bit here. The wording implies that you're toggling which services are recommended, but I was under the impression there are certain services that we recommend (which are auto-enabled) and the toggle just allows you to enable/disable those. If the latter is the case, let me know and I'll tighten up the wording.

Author: The initial setting is what "we recommend", but you can toggle it to make your own selection.
Indeed the use of the word recommended is misleading and I only used it because that is how they were called before.

So the flow is 1. you setup a subscription -> 2. you go to dashboard, recommended services are selected by default -> 3. you can adjust there as/if you want -> 4. you attach and those enabled will be auto enabled.

Reviewer: That's what I thought - I'll recommend an alternative

> Recommended services are auto-enabled when attaching a system. Which services
> are considered recommended can, for some subscriptions, be controlled on the
> [Ubuntu Pro Dashboard](https://ubuntu.com/pro/dashboard) by toggling those.

Reviewer: ```suggestion
Recommended services are auto-enabled by default when attaching a system.
You can choose which of the available services will be automatically
enabled or disabled when you attach by toggling them in the
[Ubuntu Pro Dashboard](https://ubuntu.com/pro/dashboard). 
Available services can always be enabled or disabled on the command line
with `pro enable` and `pro disable` after attaching.
```

Reviewer: Since you mentioned in the previous paragraph that the subscription controls which services are available, I think this wording is perhaps a bit clearer (and hopefully not too long)

![Toggling recommended services in the Pro Dashboard](pro-dashboard-service-toggles.png)

In case you can not control your subscription (depends on the subscription type

> and your users permissions related to it). Or if you do not want to change the

Reviewer: ```suggestion
and the user permissions granted by it). Or if you do not want to change the
```

Author: Umm, no :-)

Subscriptions differ and in them then users differ:
- free one - just defaults no permission to change via the web UI
- real subscription has some user management, you might have the permission to change things or only to see them

Your suggestion implies that the subscription would grant any permissions. That is not what I was saying. I wanted to avoid 500 more words but make it clear that (depending on complex conditions) you might or might not be able to see or tweak them.

> defaults, but you want to not auto-enable anything while attaching a particular

Reviewer: ```suggestion
defaults, but you do not want to auto-enable anything while attaching a particular
```

> machine you can always pass `--no-auto-enable` to attach, but not auto-enable

Reviewer: ```suggestion
machine you can always pass `--no-auto-enable` to `attach`, but not auto-enable
```

> any of the services.

Reviewer: ```suggestion
any services using the following command:
```

> In case you can not control your subscription (depends on the subscription type
> and your users permissions related to it). Or if you do not want to change the
> defaults, but you do not want to auto-enable anything while attaching a particular
> machine you can always pass `--no-auto-enable` to `attach`, but not auto-enable
> any services using the following command:

Reviewer: ```suggestion
If your subscription does not permit you to change the default
enabled services via the Dashboard, or if you want to keep the
defaults but do not want to auto-enable any services while attaching a particular
machine, you can pass the `--no-auto-enable` flag to `attach` using the following command:
```
